packages/core/admin/admin/src/content-manager/hooks/useLazyComponents/index.js

```diff
@@ -36,23 +36,19 @@ const useLazyComponents = (componentUids = []) => {
       setStore(Object.fromEntries(componentStore));
     };
 
-    if (componentUids.length && loading) {
+    if (loading) {
```

Comment:
```suggestion
    if (newUids.length > 0) {
    	setLoading(true)
```
I think it makes more sense to do?